pkg/util/mount/mount_linux.go

```diff
@@ -1003,10 +1003,11 @@ func (mounter *Mounter) SafeMakeDir(subdir string, base string, perm os.FileMode
 }
 
 func (mounter *Mounter) GetMountRefs(pathname string) ([]string, error) {
-	if _, err := os.Stat(pathname); os.IsNotExist(err) {
```

Comment:
does windows also need fixing since the original change also added a stat check for mount_windows?  cc @jingxu97 @andyzhangx

Reply:
looks like PathExists in the mount_helper does not consider windows OS. We could add a windows version of PathExists or add windows handling inside of PathExists?

Reply:
I am not sure which windows errors are because of corrupted mount that correspond with unix errors `ENOTCONN`, `ESTALE`, `EIO`.

Windows error list here: https://github.com/golang/sys/blob/master/windows/types_windows.go#L9

Reply:
Note that pkg syscall and sys are different

Reply:
right, pkg `sys` seems to imply that the same `syscall.Errno` codes mean different things in windows vs. unix.
And `IsCorruptedMnt` seems to be using the unix interpretation of the error codes so I would think we need a Windows specific version of `PathExists` and `IsCorruptedMnt`.

Reply:
> looks like PathExists in the mount_helper does not consider windows OS. We could add a windows version of PathExists or add windows handling inside of PathExists?

whether `PathExists` works on windows os depends on whether `IsCorruptedMnt` works on windows, it at least could compile on windows. Since the original issue is related to XFS mount issue, and windows does not support that, I think use same fix on windows would be ok.
@davidz627 could you do same fix on `windows_mount.go`? 
BTW, as I implemented `GetMountRefs` func in the beginning, you could see it's not reliable since there is no place to query all mount points on Windows, different from Linux.
https://github.com/kubernetes/kubernetes/blob/4fa5ece62ce3475d851c184bb623e6220fbc4950/pkg/util/mount/mount_windows.go#L497-L505

Reply:
i've updated the windows `GetMountRefs` as well. I believe the errcodes are different for windows though. Therefore I have removed the `IsCorruptedMnt` check as I believe it would be just checking "random" error codes and may mask real issues.

Since XFS mount issue doesn't exist on windows anyway this is not a necessary addition.